feat: #2388: load balancer documentation when using the proxy via the operaor. (#2393)
kroxylicious/kroxylicious: docs/snippets/snip-create-secret-containing-pem-certificate.adoc

> Create a secret for the certificate using a command like this:

Reviewer: ```suggestion
Create a secret for the certificate using the following command or another method:
```
